pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -436,7 +450,7 @@ func GetPipelineConditionStatus(pr *v1alpha1.PipelineRun, state PipelineRunState
 		Type:    apis.ConditionSucceeded,
 		Status:  corev1.ConditionUnknown,
 		Reason:  ReasonRunning,
-		Message: "Not all Tasks in the Pipeline have finished executing",
+		Message: fmt.Sprintf("Completed %d tasks, not start/running %d tasks, skipped %d tasks", len(successOrSkipTasks)-skipTasks, len(allTasks)-len(successOrSkipTasks), skipTasks),
```

Comment:
Suggest "Tasks Completed: %d, Incomplete: %d, Skipped: %d" here to make it a little shorter / quicker to read.

Reply:
@sbwsg  Done, thanks